### PR TITLE
VPN-2872 - Unbind navbar checked button from VPNNavigator.screen

### PR DESCRIPTION
**Pull request by brizental**

### CODE REVIEW

File: src/ui/navigator/navigationBar/VPNBottomNavigationBarButton.qml [hunk 1]
```diff
@@ -16,7 +16,7 @@ VPNIconButton {
 
     accessibleName: ""
 
-    checked: VPNNavigator.screen === _screen
+    checked: false
```
Comment (MattLichtenstein) [inline src/ui/navigator/navigationBar/VPNBottomNavigationBarButton.qml:19]: could probably just delete this line

File: src/ui/navigator/navigationBar/VPNBottomNavigationBar.qml [hunk 2]
```diff
@@ -176,5 +183,7 @@ Rectangle {
 
     Component.onCompleted: {
         root.getUnreadNotificationStatus();
+
+
```
Comment (MattLichtenstein) [inline src/ui/navigator/navigationBar/VPNBottomNavigationBar.qml:187]: extra new lines 🙅

**Review by MattLichtenstein — APPROVED** [review]
I think this is a good solution for now!

File: src/ui/navigator/navigationBar/VPNBottomNavigationBarButton.qml (hunk 1 above)
Comment (brizental) [reply src/ui/navigator/navigationBar/VPNBottomNavigationBarButton.qml:19]:
```suggestion
```

File: src/ui/navigator/navigationBar/VPNBottomNavigationBar.qml (hunk 2 above)
Comment (brizental) [reply src/ui/navigator/navigationBar/VPNBottomNavigationBar.qml:187]:
```suggestion
```

File: src/ui/navigator/navigationBar/VPNBottomNavigationBarButton.qml [hunk 3]
```diff
@@ -16,7 +16,6 @@ VPNIconButton {
 
     accessibleName: ""
```
Comment (brizental) [inline src/ui/navigator/navigationBar/VPNBottomNavigationBarButton.qml:18]:
```suggestion
```

**Pull request merged**